### PR TITLE
Feat: New Design

### PR DESCRIPTION
**Pull request by cameronmcefee**
This adds a more Sentry-themed design. The cart is fully functional, and the sumulated error is thrown when "Proceed to checkout" is clicked while items are in the cart.

### Todos

These are nice to haves, but this PR can be merged without them

- [ ] I couldn't get the sentry portion running, so I haven't checked if it actually works
- [ ] It'd be cool if we could link to the exception within Sentry in the error message

<details>
  <summary><kbd>View screenshots</kbd></summary>
  <br />
  Screenshot of store.
  Screenshot of error state.
</details>

### CODE REVIEW

**Comment by ndmanvar** [comment]
@cameronmcefee 
I'll get Sentry working on this today.

> It'd be cool if we could link to the exception within Sentry in the error message
Hmmm... I think I can do this as well (interesting idea).

File: README.md [hunk 1]
```diff
@@ -9,8 +9,6 @@ Show how Sentry works
 ## First-time Setup
 1. Install dependencies
 ```
-npm install -g serve
-npm install -g @sentry/cli
```
Comment (vudngo) [inline README.md]: @cameronmcefee This demo uses sentry-cli so we want to make sure sentry-cli is install. Is there a reason why you removed it?

Comment (vudngo) [reply README.md]: Nevermind. I see you added sentry-cli as a dependency in package.json.

Comment (cameronmcefee) [reply README.md]: It's now in package.json so we don't encourage global dependencies https://github.com/sentry-demos/react/pull/60/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R6

I cleared it with @ndmanvar before including the change.

**Pull request merged**